src/qibolab/instruments/abstract.py

```diff
@@ -20,7 +20,7 @@ def __init__(self, name, address):
         self.is_connected: bool = False
         self.signature: str = f"{type(self).__name__}@{address}"
         # create local storage folder
-        instruments_data_folder = user_folder / "instruments" / "data"
+        instruments_data_folder = INSTRUMENTS_DATA_FOLDER
```

Comment:
I suspect that this and the paths defined as attributes below can be dropped completely, as they are not used anywhere other than tests. We just need to confirm that no instruments are using them. @rodolfocarobene @Jacfomg did you use `self.tmp_folder` or `self.data_folder` anywhere in the instruments you introduced?

We could also drop them later, but it is a good opportunity to be part of this cleanup.

Reply:
I do not

Reply:
@Jacfomg is using them to store weights. See above comment.
I don't think that @rodolfocarobene is using them.

Reply:
We dont have the routine as a protocol to extract the routines for now. Could I put the weights on the repository with the runcards ?

Reply:
To me this should be fine.
What do you think @stavros11 @AleCandido?

Reply:
@AleCandido I think that @Jacfomg wants to move the weight file for zurich in qibolab_qrc_platforms.
I don't know if this is feasible. Otherwise if it works for the time being we can keep it here.

Reply:
The idea I wanted to convey should sound something like "I don't care" xD

Put wherever it works for you at the moment. In the end, they are instrument parameters, so they are conceptually part of the calibration, and so the platform runcard.
Eventually, I'd fit even them in the DB (if we have a DB with everything, I don't see the point of keeping things out of it)..